tools/multipleJobsToTschenggins.sh

```diff
@@ -29,7 +29,13 @@ function update {
 	curl --header "Content-Type: application/json" -s \
 		--request POST \
 		--data '{"cmd":"update","debug":0,"states":[{"name":"'$3'","result":"'$1'","server":"'$TARGETSERVER'","state":"'$2'"}]}' \
-		http://$BACKENDUSER:$BACKENDPASS@$BACKENDSERVER:$BACKENDPORT/tschenggins-laempli/ng/ > /dev/null
+		https://$BACKENDUSER:$BACKENDPASS@$BACKENDSERVER:$BACKENDPORT/tschenggins-laempli/  \
+        --cacert tsueri.oo.crt --insecure \
+        -A "Tschenggins-Bash-Script" \
```

Comment:
Maybe better -A "multipleJobsToTschenggins.sh/1.0" ?
To be in line with the other scripts and the typical agent string format.